projects/direct-integration-of-lighthouse-reth-and-tracing-integration-in-lighthouse.md

```diff
@@ -0,0 +1,70 @@
+# Direct(non-http) integration of Lighthouse-Reth & Tracing integration in Lighthouse
+## Motivation
+
+Stealing this part from [Eitan's project proposal](https://github.com/eth-protocol-fellows/cohort-four/blob/master/projects/lighthouse-slasher-and-beacon-node-backend.md#motivation) from cohort-four to set the groundwork.
+
+*"Now as an EPF participant I am tasked with working on a "large" project. However, based on what I've experienced so far, client developers dont just work on individual features. They frequently work on multiple tasks at the same time. Furthermore, client developers provide technical suppport to users and spend time investigating/fixing bugs."*
+
+Inspired by this I'll be taking on two projects to work on parallelly
+- Direct integration of Lighthouse-Reth (primary project)
```

Comment:
The motivation part doesn't elaborate on this at all. What is the reason to integrate lighthouse and reth to a single codebase? 
The modularity of Ethereum is a one of its strengths. Merging el and cl client would result in a hard to maintain mess of codebase with different libs and dependencies. If the motivation is to make it easier to run client as a single binary instead of doing the engine api config, I would highlight this as the motivation and take inspiration from Erigon Caplin

Reply:
Thanks a lot for the feedback ! 
added some details to it,please lmk if it looks better now or if I need to change something